docs/installation/community-operator.md

```diff
@@ -45,50 +47,74 @@ kepler-operator.v0.5.0-230815081646   Kepler    0.5.0-230815081646
+```sh
+./hack/uninstall-operator.sh -v v0.5.0-230815081646 --delete
 ```
 
 ---
+
 ## Install Kepler Community Operator from Operator Hub
 
-1. Go to Operators ❯ OperatorHub. Search for `Kepler`.
-   Click on Kepler Operator tile, then select `Continue` and then `Install`
+- Go to Operators ❯ OperatorHub. Search for `Kepler`.
+  Click on Kepler Operator tile, then select `Continue` and then `Install`
 
-![](../fig/ocp_installation/operator_installation_ocp_1_0.6.z.png)
+![](../fig/ocp_installation/operator_installation_ocp_1_0.8.z.png)
 
-2. No changes need to be made to the operator install options, select `Install`
+- Choose `dev-preview` channel for deploying `dev-preview` version of Operator and click on `Install`
 
-![](../fig/ocp_installation/operator_installation_ocp_2_0.6.z.png)
+![](../fig/ocp_installation/operator_installation_ocp_2_0.8.z.png)
 
-2. Allow the operator to install
+- Wait till Operator gets installed
 
-![](../fig/ocp_installation/operator_installation_ocp_3_0.6.z.png)
+![](../fig/ocp_installation/operator_installation_ocp_3_0.8.z.png)
 
-Follow the link to `View installed Operators in Namespace openshift-operators`
+Follow the link to view installed Operators in `openshift-operators` Namespace
 or use the UI to navigate to installed operators and select the Kepler
 Operator.
 
-3. Select `Create instance` to Create a Custom Resource for Kepler
+- Select `Create instance` to Create a Custom Resource for Kepler
+
+![](../fig/ocp_installation/operator_installation_ocp_4_0.8.z.png)
 
-![](../fig/ocp_installation/operator_installation_ocp_4_0.6.z.png)
+- There is a `Form` and `YAML` view, using the **YAML** view
+  provides more detail.
 
-4. Select `Create`. There is a `Form` or `YAML` view, using the **YAML** view
-	 provides more detail. No changes need to be made.
+![](../fig/ocp_installation/operator_installation_ocp_5a_0.8.z.png)
 
-![](../fig/ocp_installation/operator_installation_ocp_5a_0.6.z.png)
-![](../fig/ocp_installation/operator_installation_ocp_5b_0.6.z.png)
+![](../fig/ocp_installation/operator_installation_ocp_5b_0.8.z.png)
 
-5. Check that the Kepler Exporter pods are running in the `openshift-kepler-operator` namespace
+- To specify **nodeSelector** and **toleration's** for Kepler:
 
-```shell
-oc project openshift-kepler-operator
-oc get pods
+```yaml
+spec:
+exporter:
+  deployment:
+    port: 9103
+    nodeSelector:
+      foo: bar
+    tolerations:
+      - key: foo
+        operator: "Equal"
+        value: bar
+        effect: NoExecute
 ```
-A kepler exporter pod should be running on node in the cluster
+
+- Once Kepler is configured select `Create`.
+
+* Check that the Availability status of Kepler Instance should be `True`
+
+![](../fig/ocp_installation/operator_installation_ocp_6_0.8.z.png)
+
+- Check that the Kepler Exporter pods are running inside `openshift-kepler-operator` namespace
+
+```sh
+oc get pods --namespace openshift-kepler-operator
+```
```

Comment:
Rather the user should be prompter to look at `Available` condition. 
`openshift-kepler-operator` namespace is an implementation detail and should in `Developer / Design / FAQ` docs than user guide.

Reply:
`oc get kepler kepler -w` is the recommended way.